mangaki/mangaki/algo/recommendation_algorithm.py

```diff
@@ -55,17 +55,19 @@ def is_serializable(self):
         return False
 
     def save(self, filename):
-        with open(self.get_backup_path(filename), 'wb') as f:
-            pickle.dump(self, f, pickle.HIGHEST_PROTOCOL)
-        self.size = os.path.getsize(self.get_backup_path(filename)) / 1e6
+        if self.is_serializable:
```

Comment:
I'm not sure we want to silence in case of algorithms which are not serializable, why not throw an exception?

Reply:
Done!